monitoring/prometheus/generate-prom.js

```diff
@@ -43,53 +42,74 @@ scrape_configs:
     static_configs:
       - targets: ['localhost:9090']
 
- # monitor docker-compose local setups
+  # monitor data growth
 
-  - job_name: 'local-discovery-provider'
-    metrics_path: '/prometheus_metrics'
+  - job_name: 'load-test-census-stage'
+    metrics_path: '/metrics'
     static_configs:
-      - targets: ['host.docker.internal:5000']
+      - targets: ['host.docker.internal:8001']
         labels:
           host: 'host.docker.internal'
-          environment: 'remote-dev'
+          environment: 'stage'
           service: 'audius'
           component: 'discover-provider'
```

Comment:
everything has `component: 'discover-provider'` - i assume just copy-paste. also typo :)

Reply:
Oh yes! Just copy-pasting from the very first pass when all I was only focusing on was discovery-provider work.

I'll get that fixed. :)

OMG! I see the typo now! :D